pom.xml

```diff
@@ -9,10 +9,10 @@
   </parent>
 
   <artifactId>label-verifier</artifactId>
-  <version>1.5-SNAPSHOT</version>
+  <version>${revision}${changelist}</version>
   <packaging>hpi</packaging>
-  <name>Jenkins Label Verifier plugin</name>
-  <url>https://github.com/jenkinsci/label-verifier-plugin</url>
+  <name>Label verifier plugin</name>
```

Comment:
Why the change from title case?

Also https://www.jenkins.io/doc/developer/publishing/style-guides/#name

Reply:
I wasn't aware that title case was a convention.  I'll happily adjust that and remove the redundant "plugin" from the end of the name.

Reply:
Fixed in:

* #34

Reply:
> I wasn't aware that title case was a convention

I don't think it's documented or required, but it's done fairly consistently. A quick and dirty check indicates that 90+% of plugin titles with 2+ words are title case.